mypy/server/update.py

```diff
@@ -1108,3 +1109,51 @@ def target_from_node(module: str,
             return '%s.%s' % (node.info.fullname, node.name)
         else:
             return '%s.%s' % (module, node.name)
+
+
+def refresh_suppressed_submodules(
+        module: str,
+        path: Optional[str],
+        deps: Dict[str, Set[str]],
+        graph: Graph,
+        fscache: FileSystemCache) -> None:
+    """Look for submodules that are now suppressed in target package.
+
+    If a submodule a.b gets added, we need to mark it as suppressed
+    in modules that contain "from a import b". Previously we assumed
+    that 'a.b' is not a module but a regular name.
```

Comment:
Why does it need to be marked suppressed? We want it to be a real dep, right? Is this just an implementation artifact of how this is handled in a loop of processing suppressed imports?

Reply:
This just arguably makes the logic a bit simpler, as these can be handled via the same code as "real" suppressed modules. If this turns out to be bad for performance, for example, we could turn these into real deps and process these separately.